src/Compatibility/ControlGallery/src/Issues.Shared/Bugzilla32801.cs

```diff
@@ -84,7 +84,7 @@ public TestDemoPage(int level)
 
 #if UITEST && __IOS__
 		[Microsoft.Maui.Controls.Compatibility.UITests.FailsOnMauiIOS]
-		[Test]
+		//[Test]
```

Comment:
Are you sure this is the right one? We don’t run the compat project anymore?

Reply:
yeah maybe I got it wrong theres another PR for this one